src/main/archetype/ui.tests/test-module/lib/wdio-pixelmatch-service/launcher.js

```diff
@@ -0,0 +1,85 @@
+/* eslint-disable no-unused-vars */
+const PNG = require('pngjs').PNG;
+const pixelmatch = require('pixelmatch');
+const logger = require('@wdio/logger').default;
+const log = logger('PixelMatchPlugin');
+const { checkFileExists, makeDirectories, readPng, writePng } = require('./fileutils');
+const comparisonStrategy = require('./comparisonstrategy');
+const PathUtils = require('./pathutils');
+class PixelMatchPlugin {
```

Comment:
Looks like the documentation is incomplete. Could you rectify this ?

Reply:
Documentation added